src/profile-logic/symbol-store.js

```diff
@@ -6,6 +6,7 @@
 import SymbolStoreDB from './symbol-store-db';
 import { SymbolsNotFoundError } from './errors';
 import bisection from 'bisection';
+import { TextDecoder } from 'util';
```

Comment:
I think we don't want this in the application code, because this runs in the browser. That said I see that webpack just removes it... but I fear this can have some side-effects with our static checkers (eslint and flow).

So I prefer to remove it. 

Remember we inject it in the test: https://github.com/cichelero/perf.html/blob/1236501cf1fd07140b3cbdd3360f4bfc3c48b57d/src/test/store/receive-profile.test.js#L96